README.md

```diff
@@ -63,15 +63,24 @@ Usage example:
         tune.exfat -l /dev/sda1
     2. set new volume label.
         tune.exfat -L "new label" /dev/sda1
+    3. print current volume serial.
```

Comment:
tune2fs prints all parameters of superblock with -l option as below.

> tune2fs -l /dev/sda1
Filesystem volume name:   <none>
Last mounted on:          /
Filesystem UUID:          9d13d129-4879-4efd-bb03-4fa8daa36fa8
Filesystem magic number:  0xEF53
....

What do you think about it?

Reply:
Well, I don't think that It is necessary to match the options with tune2fs. We need to match the options of the existing exfat-utils or dosfstools for compatibility issues. but not ext4 and f2fs tools. They are just needed for reference.

Reply:
I agreed, But I mean that we don't need to implement options to get each parameter.  instead of it, we can implement the option like "-l" to get all parameters.

Reply:
Performing two actions in one option makes the implementation weird.

The implementation is cleared if you use each of the two options for get/set serial with the following require/no_argument in struct option.
static struct option opts[] = {
...
        {"print-serial",        no_argument,            NULL,   'i' },
        {"set-serial",          required_argument,      NULL,   'I' },

And working two action in -I option, if there is no input with -I (tune.exfat -I /dev/sda1), optarg will be "/dev/sda1".
And the case of volume label was already merged into #master, which is an implementation divided into two options(-l and -L). If users looks sane, Let's check it that time again. What do you think ?

Reply:
Okay, but I am worried that code becomes a little complicated because we need to implement get/set options for each parameters.